testsuite/features/secondary/srv_restart.feature

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2021 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Restart the spacewalk-services via UI
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+    When I follow the left menu "Admin > Manager Configuration > Restart"
+    And I check "restart"
+    And I click on "Restart"
+    And I wait for "300" seconds
```

Comment:
We generally try to avoid those fixed duration waits. They make lose time and are error-prone.

Please wait on something testable, like the thing on next line.

Simply Remove line 11?

Reply:
Yes I understand, but its a mentioned in the UI that it needs 5 min to restart and refresh so I dont do it to just wait, I do this because that the way the restart from the webui works

Reply:
This is the message you get when you restart through the webUI .
"SUSE Manager restarting. If this page has not refreshed in 5 minutes please click here to try manually."

Reply:
But the reboot use to take much less, and time is gold in our automated tests.

Reply:
And in any case, why wait a fix amount when you can test for a precise event? :-)

Reply:
I refactored it. Now there is no explicit wait. I think its ok